src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/repository/AgencyVisitTimeRepository.kt

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.repository
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.AgencyLocation
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.AgencyVisitTime
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.AgencyVisitTimeId
+import java.time.LocalTime
+
+@Repository
+interface AgencyVisitTimeRepository :
+  JpaRepository<AgencyVisitTime, AgencyVisitTimeId> {
+
+  fun findByStartTimeAndAgencyVisitTimesId_WeekDayAndAgencyVisitTimesId_Location(
+    startTime: LocalTime,
```

Comment:
I'm impressed this works 😎 given the actual columns seemed to have a date element that needs to be ignored

Reply:
to be tested on oracle